docs/data/maturity.json

```diff
@@ -19,7 +19,25 @@
     "area": "Build",
     "feature": "Cloud Native Buildpacks support",
     "maturity": "GA",
-    "description": "Skaffold natively support for artifacts built with Cloud Native Buildpacks"
+    "description": "Skaffold natively support for artifacts built with Cloud Native Buildpacks",
```

Comment:
nit I think this `Skaffold natively supports artifacts built with Cloud Native Buildpacks`

(This isn't related to your changes but might be easy to slip in)

Reply:
Updated.